core/internal/testdata/sleep_job.json

```diff
@@ -0,0 +1,9 @@
+{
+  "initiators": [
+    {"type": "web"}
+  ],
+  "tasks": [
+    {"type": "sleep"},
+    {"type": "noop"}
```

Comment:
Let's [fix this](https://www.pivotaltracker.com/story/show/166400993) in another PR.